<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SLAMViewer</class>
 <widget class="QMainWindow" name="SLAMViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SLAMViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateSetting">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Setting</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbnOnline">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Online</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbnFile">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPcapFile">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>64</y>
      <width>59</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxINSSize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>INS Cache size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtINSSize">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btnChoosePcapFile">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPcapFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="Choose .pcap file"/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxHDLSize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HDL Cache size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtHDLCache">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxSaveINS">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>220</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save INS to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSaveINS">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btnSaveINSTo">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>220</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxSaveHDL">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>252</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save HDL to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSaveHDL">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btnSaveHDLTo">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>250</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtCalibFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btnChooseCalibFile">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnChooseINSFile">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtINSFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
